n3fit/src/evolven3fit_new/cli.py

```diff
@@ -0,0 +1,236 @@
+from . import evolve, utils
```

Comment:
I don't remember who asked, but I was asked esplicitly to do exactly the opposite (which is why now everything is inside `cli.py`). For me it is fine to move the second time but please avoid me the third :).

Reply:
I don't know if it is what you refer to, but some time ago, I asked you to write the CLI as a very small wrapper around an API (for Pineko).
That idea is that everything you do in bash, you'd like to be able to control programmatically also from Python (it is useful if you want to automate some tasks). Moreover, it is easier also for testing: if you have a large bunch of code in the CLI only, you have to test it with bash calls, rather than Python ones.

Maybe it is not what you refer to, but in case, this does not apply to this file: there are only two elements, a global value and a single function. The function is only making two calls, so it is mainly about its documentation. If you are able to put the documentation in another suitable place, it is just redundant, since it is used in a single place. And if you move also the value somewhere else (`utils.py`? or whatever) you can also get rid of one file :)

Reply:
I am actually referring to your comment of 12 days ago on this PR:

> Consider also to move the CLI script definition inside the package (it is more integrated) and just call a package function in the script.

However, again, it is not a problem: I can move again :)

Reply:
> > Consider also to move the CLI script definition inside the package (it is more integrated) and just call a package function in the script.

Of this I'm still convinced, for the same reasons of Pineko. What I meant here was to reduce all subcommands to a single function call, and keep the script parser only. Being me, I would also define the parsers in the package, since scripts are much more annoying to test and maintain in general - but this is definitely personal preference, in the end is just a ~100 lines script, with only parsers.
But the `cli.py` only contains a single function, with two calls: if it makes sense as an operation, you can keep and move in another existing module. Otherwise, if it is only a wrapper for two function calls, called that way only once, you can inline in the script, also because the other branch is already not a single function call:
https://github.com/NNPDF/nnpdf/blob/841c123d1fa3f3f50c518df28718578115e40f22/n3fit/src/n3fit/scripts/evolven3fit_new.py#L122-L140

Reply:
Baseline: modularize as much as possible, to organize things, but not too much. If you have a module for a function, and a function with the calls (called only once), you can probably simplify a bit, and group with something else.

Reply:
At the end this solution seemed to me good enough: I would not like to put both the `xgrid` and the function inside the script. If you really have a strong opinion about that for me it is also fine to change, otherwise I would rather keep as it is.